storybookjs/storybook
# Addon Info: Omit empty inline info header

Issue: Creating stories using `addWithInfo`, configured not to show headers, and to render info inline displays a blank header container on each story.

## What I did
Instead of unconditionally rendering the header container and filling it with the (possibly `null`) result of `_getInfoHeader()`, I check the result first, and render the container conditionally.

## How to test
Check the `'simple usage (no header)'` example. Before this change is applied, it appears with a large empty white area above the example.
<img width="831" alt="screen shot 2017-06-17 at 2 35 02 pm" src="https://user-images.githubusercontent.com/7367/27256462-330a7c68-536a-11e7-851f-fc8fdd442561.png">


After this change is applied, the empty white area is absent.
<img width="829" alt="screen shot 2017-06-17 at 2 32 22 pm" src="https://user-images.githubusercontent.com/7367/27256452-d477d722-5369-11e7-8cdd-e0b8bad61077.png">
